compiler-core/src/erlang/tests/strings.rs

```diff
@@ -438,3 +438,32 @@ pub fn main(x) {
 "#,
     );
 }
+
+#[test]
+fn assert_const_concat() {
+    assert_erl!(
+        r#"
+const foo = "foo"
+const bar = foo <> "bar"
+
+pub fn main() {
+  bar
+}
+"#
+    );
+}
+
+#[test]
+fn assert_const_concat_multiple() {
+    assert_erl!(
+        r#"
+const foo = "foo"
+const foobar = foo <> "bar"
+const foofoobarbaz = foo <> foobar <> "baz"
```

Comment:
Split this into multiple tests please, we only test one example per test.

Could we also have a test for a much larger number of concatenations in one constant too please.